packages/agoric-cli/src/deploy.js

```diff
@@ -298,29 +305,26 @@ export default async function deployMain(progname, rawArgs, powers, opts) {
           );
         }
 
-        const cache = makeCache(
-          E(E(E.get(bootP).wallet).getBridge()).getCacheCoordinator(),
-        );
-
         let cachedLeader;
         const makeDefaultLeader = async leaderOptions => {
-          if (cachedLeader === undefined) {
-            const conn = await getDefaultConnection();
-            const { type, rpcAddresses } = conn;
-            assert.equal(
-              type,
-              'chain-cosmos-sdk',
-              X`${type} doesn't support casting followers`,
-            );
-            cachedLeader = makeLeaderFromRpcAddresses(
-              rpcAddresses,
-              leaderOptions,
-            );
+          if (cachedLeader) {
+            return cachedLeader;
           }
+          const conn = await getDefaultConnection();
+          const { type, rpcAddresses } = conn;
+          assert.equal(
```

Comment:
```suggestion
          type === 'chain-cosmos-sdk' ||
            Fail`${type} doesn't support casting followers`;
```
I suspect there's no reason to redact `type`. If so
```suggestion
          type === 'chain-cosmos-sdk' ||
            Fail`${q(type)} doesn't support casting followers`;
```
though neither of these need to be done in this PR.